src/app/screens/Onboard/ConnectLnd/index.tsx

```diff
@@ -52,6 +53,7 @@ export default function ConnectLnd() {
       // TODO: for native connectors we currently skip the validation because it is too slow (booting up Tor etc.)
       if (account.connector === "nativelnd") {
         validation = { valid: true, error: "" };
+        browser.permissions.request({ permissions: ["nativeMessaging"] });
```

Comment:
this returns a promise with true/false if the permissions were granted or not. 
I think we should fail if the user does not provide the permissions.

Reply:
maybe something:
```js
browser.permissions.request({ permissions: ["nativeMessaging"] }).then(hasPermissionGranted => { 
  if (hasPermissionGranted) { 
    validation = { valid: true, error: "" };
  } else {
    validation = { valid: false, error: "Native permissions are required to connect through Tor." };
  }
});
```